src/api.py

```diff
@@ -196,19 +198,39 @@ def recitation(self):
                 else:
                     raise EditorNotFound()
             except EditorNotFound:
-                print("Text editor not found; set the 'EDITOR'" +
+                if platform.system() == "Windows":
+                    editor = "notepad"
+                    windowsfilename = f"{self.target.reference}"
+                    windowsfilename = windowsfilename.replace(":","-")
```

Comment:
To prevent any ambiguity, I think substituting ";" for ":" is better than "-". A dash can be used when reciting a range of verses, so this change can lead to confusion.
```suggestion
                    windowsfilename = windowsfilename.replace(":",";")
```